Src/library/memory_analyzer/ma_draw_history.e

```diff
@@ -15,6 +15,12 @@ inherit
 		export
 			{NONE} all
 		end
+
```

Comment:
I didn't want to change too much the current design.
I just changed it for one interface that was using a lot the main_window.

Now I can do that for all callers of main_window, but this means change the related attribute to be pseudo once per object (I mean using internal_foo: detachable like foo ..., and foo: FOO do ....-- use internal_foo end)